docker/development/app/Dockerfile

```diff
@@ -11,7 +11,6 @@ ENV RAILS_ENV='' \
     POSTGRES_HOST='' \
     POSTGRES_PASSWORD='' \
     POSTGRES_USER='postgres' \
-    POSTGRES_DB='' \
```

Comment:
An interesting caveat, if `POSTGRES_DB` is set - in this case to the empty string, [the `database.yml` uses its value rather than the default](https://github.com/rubyforgood/mutual-aid/blob/main/config/database.yml#L40). In this case when we remove the default value of `POSTGRES_DB` from the development `docker-compose.yml`, the Rails app looks for the database named `''` rather than the expected defaults of `mutualaid_development` or `mutualaid_test` depending on the environment. Furthermore, even if the value is set here to the empty value - IE `POSTGRES_DB=`, the same behavior persists. Therefore, I decided to remove this `ENV` declaration so that the expcted defaults of the `database.yml` would be used.